examples/docs/en-US/layout.md

```diff
@@ -276,6 +276,40 @@ Flexible alignment of columns.
 ```
 :::
 
+### Responsive Layout
+
+Referring to the Bootstrap responsive design, here preset four dimensions：`xs` `sm` `md` `lg`.
```

Comment:
Taking example by Bootstrap's responsive design, four breakpoints are preset: `xs`, `sm`, `md` and `lg`.